package.json

```diff
@@ -22,8 +22,9 @@
     "bundle": "rm -rf dist && babel src --out-dir dist",
```

Comment:
This should be changed to `parcel` right?

Reply:
No. Parcel is only used to bundle the example application. The module is still bundled using Babel.